src/argus_htmx/templates/registration/login.html

```diff
@@ -18,8 +19,16 @@ <h2 class="card-title">Log In</h2>
         </div>
       </label>
       {% endfor %}
-      <button type="submit" class="btn btn-primary">Log In</button>
+      <button type="submit" class="btn btn-primary">{{ backends.username_password.button }}</button>
     </form>
+    {% endif %}
+
+    {% if backends.link %}
+    <h3>Login with</h3>
+    {% endif %}
```

Comment:
nitpick, while not impacting any functionality, it reads more natural if you put the endif after the endfor